proksee/assembly_measurer.py

```diff
@@ -80,7 +80,8 @@ def measure_quality(self):
         if not os.path.exists(self.contigs_filename):
             raise FileNotFoundError("File not found: " + self.contigs_filename)
 
-        quast_command = "quast --contig-thresholds 0," + str(self.minimum_contig_length) + " " + \
+        quast_command = "quast --contig-thresholds 0," + str(self.minimum_contig_length) + \
```

Comment:
Is `minimum_contig_length` always guaranteed to be a positive integer? Or is it possible someone could submit an arbitrary string here, which would either cause `quast` to crash, or run additional commands?

In other words, are you escaping/sanitizing the value of `minimum_contig_length` someplace?

Reply:
Updated click to ensure that the minimum contig length, number of threads, and amount of memory are enforced to be integers, but further work needs to be done: #81

Reply:
Also updated to use click integer ranges where appropriate.